### PR TITLE
Fix: define error type

### PR DESCRIPTION
**Pull request by EMazzaglia**
## Pull request type

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?
The application was not starting correctly because of a missing type declaration for an error. 

Issue Link: N/A

## What is the new behavior?

The type declaration was included
Removed unnecessary parameters on docker config

## Other information

N/A

## Preview

N/A

### CODE REVIEW

File: src/services/session.service.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export class SessionService {
     try {
       this.userService.hashUserPassword(user);
       return await this.userRepository.save(user);
-    } catch (error) {
+    } catch (error: any) {
```
Comment (pablanco) [inline src/services/session.service.ts]: could we try to not use `any`? Maybe `unknown` or custom type would be a better approach

Comment (EMazzaglia) [reply src/services/session.service.ts:26]: Yes, totally agree, I replaced within `unknown`

Comment (EMazzaglia) [reply src/services/session.service.ts]: As we discussed, I removed the type and throwed a custom exception without using any information of the error.

**Review by pablanco — APPROVED** [review]
Nice Job!

**Review by Alxzu — APPROVED** [review]

**Pull request merged**